src/request-api.js

```diff
@@ -23,16 +23,16 @@ function onRes (buffer, cb) {
       return cb(err)
     }
 
-    const stream = !!res.headers['x-stream-output']
-    const chunkedObjects = !!res.headers['x-chunked-output']
+    const stream = Boolean(res.headers['x-stream-output'])
+    const chunkedObjects = Boolean(res.headers['x-chunked-output'])
     const isJson = res.headers['content-type'] && res.headers['content-type'].indexOf('application/json') === 0
 
     if (res.statusCode >= 400 || !res.statusCode) {
       const error = new Error(`Server responded with ${res.statusCode}`)
 
-      return Wreck.read(res, {json: true}, (err, payload) => {
-        if (err) {
-          return cb(err)
+      return Wreck.read(res, {json: true}, (err1, payload) => {
+        if (err1) {
```

Comment:
I like shadowing. I think it is a cool feature of a language that you can redefine vars under narrower  scopes